src/java/org/apache/cassandra/tcm/migration/CMSInitializationRequest.java

```diff
@@ -109,12 +109,12 @@ public long serializedSize(CMSInitializationRequest t, int version)
     public static class Initiator
     {
         public static final Serializer serializer = new Serializer();
-        public final InetAddressAndPort initiator;
+        public final InetAddressAndPort endpoint;
```

Comment:
can we annotate `@Nullable`?  Its null here `org.apache.cassandra.tcm.migration.Election#MIGRATING`